pom.xml

```diff
@@ -46,7 +46,7 @@ under the License.
     <changelist>999999-SNAPSHOT</changelist>
     <jenkins.version>2.303.3</jenkins.version>
     <java.level>8</java.level>
-    <jenkins-test-harness.version>1498.v53acb0fd4634</jenkins-test-harness.version>
```

Comment:
this was changed a few months ago due issues with the dependencies of the jenkins test harness, the issue seems resolved

Reply:
> the issue seems resolved

Excellent!